sdk/storage/Azure.Storage.Blobs.ChangeFeed/tests/ShardTests.cs

```diff
@@ -70,7 +70,7 @@ public async Task GetCursor()
                 It.IsAny<long>(),
                 It.IsAny<long>(),
                 It.IsAny<CancellationToken>()))
-                .Returns((bool _, string path, long _, long _, CancellationToken _) => Task.FromResult(chunks[path].Object));
+                .Returns((bool _, string path, long __, long ___, CancellationToken ____) => Task.FromResult(chunks[path].Object));
```

Comment:
Did it not compile? I recall having same problem. Bumping installed .NET SDK solved that for me.
Btw. The multiple _ look funny.

Reply:
This feature (lambda parameter discard) is a preview compiler feature.